### PR TITLE
Added changes related Schema evolution and E2E tests

### PR DESCRIPTION
**Pull request by ashisprasad06**

### CODE REVIEW

File: ci/jobs/development.yml [hunk 1]
```diff
@@ -2,13 +2,23 @@ jobs:
...
         trigger: true
-      - .: (( inject meta.plan.terraform-bootstrap ))
-      - .: (( inject meta.plan.terraform-apply ))
-      - .: (( inject meta.plan.terraform-plan ))
+      - get: dataworks-behavioural-framework
+        trigger: false
+      - put: meta
+        resource: meta-development
+    - .: (( inject meta.plan.terraform-bootstrap ))
+    - .: (( inject meta.plan.terraform-apply ))
```
Comment (steveburton4) [inline ci/jobs/development.yml:17]: Can you add this to the plan and apply steps please? I know they are the defaults but I think the explicitness is good:

``` 
params:
          TF_WORKSPACE: 'default'
```

File: ci/jobs/qa.yml [hunk 2]
```diff
@@ -8,10 +8,20 @@ jobs:
...
       - .: (( inject meta.plan.terraform-apply ))
         params:
           TF_WORKSPACE: 'qa'
       - .: (( inject meta.plan.terraform-plan ))
         params:
           TF_WORKSPACE: 'qa'
+      - .: (( inject meta.plan.run-e2e-tests ))
+        config:
+          params:
+            TF_WORKSPACE: 'qa'
```
Comment (steveburton4) [inline ci/jobs/qa.yml:25]: You don't need the workspace param for this step

File: ci/jobs/development.yml (hunk 1 above)
Comment (ashisprasad06) [reply ci/jobs/development.yml:17]: Thanks, required changes

File: ci/jobs/qa.yml (hunk 2 above)
Comment (ashisprasad06) [reply ci/jobs/qa.yml]: removed the workspace

**Review by steveburton4 — APPROVED** [review]

**Pull request merged**